Gemfile

```diff
@@ -1,45 +1,42 @@
-source 'https://rubygems.org'
+#
+# The default setup of Huginn is pretty large and may not fit on
+# servers with lower RAM.  To conserve RAM, you can turn off some of
+# the Agent types you don't need and avoid loading their dependencies.
+# It is also a good idea to not install gems only for development on
+# production.  To do this, list the gem groups you want to opt out of
+# after `--without` when you run `bundle` on your running environment.
+# e.g.:
+#
+#     bundle install --deployment --without development test basecamp human_task jabber mqtt weibo
```

Comment:
If people run this locally, they may get confused about how to also use development mode.  Perhaps we need a short tutorial on how to run Huginn in different modes?